guides/v2.1/cloud/reference/docker-config.md

```diff
@@ -61,17 +63,17 @@ The `ece-tools` package provides a `docker:build` command to generate the Docker
     ```bash
     vendor/bin/ece-tools docker:config:convert
     ```
-    
+
     This converts your `.php` files to `.env` configuration files.
-    
+
     * `docker/config.env`
     * `docker/global.env`
 
 1. Build files to containers and run Docker in the background.
 
     ```bash
     docker-compose up -d --build
-    ``` 
+    ```
 
 1. Install Magento in your Docker environment.
```

Comment:
The final step is missing.
Need the step explaining they can view in the browser.